src/utils/lm/install.js

```diff
@@ -0,0 +1,262 @@
+    checkLFSFiltersStep(async (ctx, task, installed) => {
+      if (!installed) {
+        await execa('git', ['lfs', 'install'])
+        task.title += chalk.dim(' [installed]')
+      }
+    }),
+    getSetupStep(skipInstall),
+    setupGitConfigStep,
+  ]
+
+  const tasks = new Listr(steps)
+  await tasks.run()
+
+  return !skipInstall
+}
+
+const skipHelperInstall = function (skip) {
+  if (skip) {
+    return `Netlify's Git Credential Helper already installed with a package manager`
+  }
+}
+
+const installedWithPackageManager = async function () {
+  const installed = hasbin.sync('git-credential-netlify')
+  return installed && !(await fileExistsAsync(getBinPath()))
+}
+
+const setupWindows = function (skipInstall) {
+  return {
+    title: `Installing Netlify's Git Credential Helper for Windows`,
+    skip: () => skipHelperInstall(skipInstall),
+    task: installWithPowershell,
+  }
+}
+
+const installUnixHelper = async function () {
+  const binPath = getBinPath()
+  const shouldFetch = await shouldFetchLatestVersion({
+    binPath,
+    packageName: PACKAGE_NAME,
+    execArgs: ['version'],
+    pattern: `${PACKAGE_NAME}\\/v?([^\\s]+)`,
+    execName: EXEC_NAME,
+  })
+  if (!shouldFetch) {
+    return
+  }
+
+  await fetchLatestVersion({
+    packageName: PACKAGE_NAME,
+    execName: EXEC_NAME,
+    destination: binPath,
+    extension: 'tar.gz',
+  })
+}
+
+const setupUnix = function (platformName, skipInstall) {
+  return {
+    title: `Installing Netlify's Git Credential Helper for ${platformName}`,
+    skip: () => skipHelperInstall(skipInstall),
+    task: async () => {
+      await installUnixHelper()
+      await setupUnixPath()
+    },
+  }
+}
+
+const installWithPowershell = async function () {
+  const script = `[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+iex (iwr -UseBasicParsing -Uri https://raw.githubusercontent.com/netlify/netlify-credential-helper/53fbf6e2b82d1ae366d096a24022fa259d19329a/resources/install.ps1)`
+
+  const temp = await mkdtempAsync(path.join(tempDir, 'netlify-'))
+  const scriptPath = path.join(temp, 'install.ps1')
+
+  await writeFileAsync(scriptPath, script)
+
+  return await execa('powershell', ['-ExecutionPolicy', 'unrestricted', '-File', scriptPath, '-windowstyle', 'hidden'])
+}
+
+const isBinInPath = function () {
+  const envPath = process.env.PATH || ''
+  const binPath = getBinPath()
+  return envPath
+    .replace(/"+/g, '')
+    .split(path.delimiter)
+    .some((part) => part === binPath)
+}
+
+const setupUnixPath = async function () {
+  const shellInfo = shellVariables()
+
+  if (isBinInPath()) {
+    return true
+  }
+
+  const initContent = `
+# The next line updates PATH for Netlify's Git Credential Helper.
+if [ -f '${shellInfo.path}' ]; then source '${shellInfo.path}'; fi
+`
+
+  switch (shellInfo.shell) {
+    case 'bash': {
+      const bashPath = `script_link="$( command readlink "$BASH_SOURCE" )" || script_link="$BASH_SOURCE"
```

Comment:
Should this be inlined in a sibling Bash file (then `fs.readFile()` used on it)? 
This would make it easier to read, and provide with syntax highlighting.

Reply:
Sure, that makes sense. I'll change it

Reply:
Done in https://github.com/netlify/cli/pull/1489/commits/f693f79abbb3d0d38313f435ffdebcd542800c17